Neos.ContentRepositoryRegistry/Classes/Factory/EventStore/DoctrineEventStoreFactory.php

```diff
@@ -24,8 +24,8 @@ public function build(ContentRepositoryId $contentRepositoryIdentifier, array $c
         );
```

Comment:
Was it intentional that you didn't update the argument `$contentRepositoryIdentifier` of the `build` function here and in the interface?

Reply:
thanks, fixed :)